tests/runners/airflow/build/http/python/test_python_server_image_builder.py

```diff
@@ -50,9 +50,9 @@ def test_build_python_server(self):
     def test_build_python_server_with_pip_conf(self):
         build_out = self.__test_build_python_server(use_pip_conf=True)
 
-        self.assertTrue(
```

Comment:
did you mean to comment this out? why?

Reply:
It doesn't align with what we have in NI.
So either we need it or not. @aviemzur @zionrubin